Dockerfile

```diff
@@ -53,4 +53,4 @@ CMD ["run"]
 
 # https://docs.docker.com/engine/reference/builder/#healthcheck
 HEALTHCHECK --interval=1m --timeout=3s \
-    CMD curl -f http://localhost:8000/health || exit 1
+    CMD curl -f http://localhost:8100/health || exit 1
```

Comment:
This healthcheck is used in production too and the production port is set in the gunicorn config. The nginx server also reverse proxies to the port set in gunicorn so we would need to update those in the deployment project too.

Reply:
I'll look at that in the deployment repo.

Reply:
Rather than trying to tweak the deployment, I've changed the repo here to only expose 8100 as part of the `docker-compose.yml` file.  As far as I can tell, a completely different `docker-compose` file is used during deployment so there shouldn't be any conflict.  Please advise if that's not the case.  Developers using Docker for Crawler will get the benefit without impacting the deployment to UAT or production.